app/views/schools/transferring_participants/check_answers.html.erb

```diff
@@ -0,0 +1,85 @@
+<% content_for :title, "Check your answers" %>
+
+<% content_for :before_content, govuk_back_link(text: "Back", href: :back) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <span class="govuk-caption-xl"><%= @school.name %></span>
+    <h1 class="govuk-heading-xl">Check your answers</h1>
+
+    <%= govuk_summary_list do |summary_list| %>
+      <% summary_list.row do |row| %>
+        <% row.key { "Name" } %>
+        <% row.value { @transferring_participant_form.formatted_full_name } %>
+        <% row.action(text: :none) %>
+      <% end %>
+
+      <% summary_list.row do |row| %>
+        <% row.key { "TRN" } %>
+        <% row.value { @transferring_participant_form.trn } %>
+        <% row.action(text: :none) %>
+      <% end %>
+
+      <% summary_list.row do |row| %>
+        <% row.key { "Date of birth" } %>
+        <% row.value { @transferring_participant_form.date_of_birth.to_date.to_s(:govuk) } %>
+        <% row.action(text: :none) %>
+      <% end %>
+
+      <% summary_list.row do |row| %>
+        <% row.key { "Joining date" } %>
+        <% row.value { @transferring_participant_form.start_date.to_date.to_s(:govuk) } %>
+        <% row.action(text: "Change",
+                      visually_hidden_text: "joining date",
+                      href: url_for({ action: :teacher_start_date})) %>
+      <% end %>
+
+      <% summary_list.row do |row| %>
+        <% row.key { "Email address" } %>
+        <% row.value { @transferring_participant_form.email } %>
+        <% row.action(text: "Change",
+                      visually_hidden_text: "email",
+                      href: url_for({ action: :email})) %>
+      <% end %>
+
+      <% if @transferring_participant_form.mentor_id %>
+        <% summary_list.row do |row| %>
+          <% row.key { "Mentor" } %>
+          <% row.value { @transferring_participant_form.mentor.full_name.titleize } %>
```

Comment:
If the assign later is selected i think we need some text to say that here

Reply:
I thought the validation set this to `nil` when the assign mentor later was selected, I thought you'd done that for a reason?

Reply:
Just to handle it when we set the mentor. I was thinking to just show the row if the person being transferred is an ect and then show the name if its not blank, if its blank "assign later"